### ISSUE TITLE
GPX 1.0 vs 1.1

### ISSUE THREAD

**Issue by Phyks** [pdhoopr/gps-to-gpx#11]
Hi,

Actually, I've just noticed the current version of the lib output a GPX 1.1 file (which is latest version). This is perfectly fine and works well except for the recently added `course` field. Actually, `course` is defined in GPX 1.0 spec: http://www.topografix.com/gpx_manual.asp.

However, it seems to have disappeared from the GPX 1.1 spec: https://www.topografix.com/gpx/1/1/.

Not sure what is best to do here, either:

* Leave it as is, with `course` element in GPX 1.1 which might not be valid.
* Move to a plain GPX 1.0 file rather than GPX 1.1 (I think there is only the header to change, not 100% sure though), to support `course`.
* Drop support for `course`.

Best,

**Comment by pdhoopr** [comment]
Good catch! Actually, looking at the schema, both `course` and `speed` are no longer standard in GPX 1.1. However, v2 of Garmin's trackpoint extensions schema allow for both elements to be included as extensions. So, I think the best approach is the following:

- Drop support for `course` and `speed` from the library
- Update the schemas on the root `gpx` element (I'll post the desired changes at the end of this comment)

The ability to pass trackpoint extensions to the library already exists, so there shouldn't be anything else needed there. When using the library, though, you will have to update your code to pass `course` and `speed` as extensions instead.

**GPX schemas**
The following namespace/schema attributes should be set on the opening `<gpx>` tag as follows:

- `xmlns="http://www.topografix.com/GPX/1/1"`
- `xmlns:gpxx="http://www.garmin.com/xmlschemas/GpxExtensions/v3"`
- `xmlns:gpxtpx="http://www.garmin.com/xmlschemas/TrackPointExtension/v2"`
- `xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"`
- `xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd http://www.garmin.com/xmlschemas/GpxExtensions/v3 http://www.garmin.com/xmlschemas/GpxExtensionsv3.xsd http://www.garmin.com/xmlschemas/TrackPointExtension/v2 http://www.garmin.com/xmlschemas/TrackPointExtensionv2.xsd"`

Would you like to open a PR implementing these changes? If not, I'll take care of them when I have a chance.

**Comment by pdhoopr** [comment]
Actually, now that the `course` and `speed` elements have been introduced, I'd consider it a breaking change to remove them, even if they aren't valid. Let's leave them for now and I'll take remove them completely in v2.

Updating the schemas and switching your code to use `course` and `speed` as extensions should be fine for now.
